Gemfile

```diff
@@ -65,6 +67,7 @@ group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem 'capybara'
   gem 'selenium-webdriver'
+  # gem "factory_bot", "~> 6.4"
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly add  `gem 'tzinfo-data'` to your `Gemfile` because it's problem to check your `spec` files 🙏🏼 

![image](https://github.com/GeekyHacks/FlightOperationsBlog/assets/63915024/2627f0a6-56ce-435c-b50e-a930e0fa5139)